org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/correction/ModifierCorrectionsQuickFixTest.java

```diff
@@ -1203,11 +1203,11 @@ public void testClassExtendFinalClass() throws Exception {
 
 	@Test
 	public void testAddSealedMissingClassModifierProposal() throws Exception {
-		Map<String, String> options21 = new HashMap<>();
```

Comment:
I think sealed classes were finalized in Java 17, although I think we added tests for them in the Java 19 release. Can these tests work without the preview feature enablement and just Java 17 ?

Reply:
Fixed.

Reply:
@snjeza wait, I don't think this is fixed. Have a look at `testAddSealedMissingClassModifierProposal`, `testAddSealedAsDirectSuperClass`, `testAddPermitsToDirectSuperClass`, `testTypeInSealedTypeDeclaration`. Don't they all us Java 22 with preview features on ? Why not just have them set VERSION_17 and remove the preview feature options (should be off by default?)

Reply:
Fixed